gridsome.client.js

```diff
@@ -77,26 +89,22 @@ export default function (Vue, options, { appOptions, router }) {
   router.beforeResolve((to, from, next) => {
     // do not rewrite build paths
     if (process.isServer) {
-      next()
-      return 
+      return next()
     }
 
     // if option is disabled skip whole logic
     if (options.enablePathRewrite === false) {
```

Comment:
I tried changing this option and setting `enablePathRewrite: false` in my site settings but paths were still changed and prefixed when switching locale in my page (by clicking a flag, for example). Maybe I didn't understand the behaviour, I'm not sure

Reply:
That option will skip a route redirect used to keep the translated parts in path when you don't use the `$tp` for all links in page. Is not strictly related to language switch (are you switch locale from `$i18n` or just using a path redirect?) but with other liinks.

For example you have an homepage with these links:
```html
<g-link to="/blog"> Blog </a>
<g-link to="/who-we-are"> Who we are </a>
```
and you're not using `$tp` to translate it. When you click on the link the `beforeResolve` it checks the current locale and perform a redirect if `/en/`, `/de/`  if not present.

When you switch of `enablePathRewrite` and you click on these links the path remain untouched and basically use the fallback locale. 

Not sure I healed your doubt :sweat_smile:

Reply:
Ok I get it now! Thank you for the explanation :)